gqcp/tests/CMakeLists.txt

```
@@ -1,6 +1,6 @@
 set(test_target_sources)
 
-add_subdirectory(Basis)
+#add_subdirectory(Basis)
```

```suggestion
add_subdirectory(Basis)
```

Could you re-enable these tests?